client/src/components/date/datepicker.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import DayPicker, { DateUtils } from 'react-day-picker';
+import 'react-day-picker/lib/style.css';
+
+const ReactDayPicker = ({ handleUpdate, disabledDays, canChangeMonth }) => {
+  const [days, setDate] = useState([]);
+
+  /*
+  useEffect(() => {
+    if (resetDays && !updating) {
+      setDate([]);
+    };
+
+    resetDays, updating
+
+    if (updating) {
+      const newArray = updating.map((day) => new Date(day));
+      setDate(newArray);
+    }
+    
+  }, [resetDays, updating]);
+  */
+  
+  const handleDayClick = (day, { selected }) => {
```

Comment:
Do we want to disallow the user from selecting a date in the past? Right now we don't but since the "next due date" always just overrides the displayable "Due Date" I think it's worth just restricting the user from selecting a date/dates in the past

Reply:
I choose to allow the user to select days from the past. For selecting days of the month, I wanted them to be able to select any day in the current month, instead of having to select to go to the next month, and then choose a day in the month. Also, if it was the last day of the month, and they see a calendar that has one day available, and the other days disabled, it doesn't look that good to me. Let me know what you think.

Reply:
I see. Yeah this is fine I guess, no biggie